### PR TITLE
Add vm-import-operator blog post

### PR DESCRIPTION
**Pull request by machacekondra**
**What this PR does / why we need it**:

**Does this PR fix any issue?** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
> Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by machacekondra** [comment]
@pkliczewski  Initial work, just so we agree on structure. Do you think we should add some more sections, like features, possible mappings, etc, or just keep it short and simple?

File: _posts/2020-08-06-Import-VM-from-oVirt.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+comments: true
+title: Import virtual machine from oVirt
+pub-date: August 07
+pub-year: 2020
+---
+
+## About vm-import-operator
+Virtual machine import operator makes a easier life to users who want to migrate their virtual machine workload from different infrastructures to kubevirt. Currently the operator support only migration from oVirt. Operator is configurable so user can define how the storage or network should be mapped. For the disk import the vm import operator is using the CDI, so in order to have the vm import working you must have both kubevirt and CDI deployed.
+
+## Setup vm-import-operator
```
Comment (pkliczewski) [inline _posts/2020-08-06-Import-VM-from-oVirt.md:54]: We may add that HCO installs it by default.

File: _posts/2020-08-06-Import-VM-from-oVirt.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+Source code for virtual machine import operator is hosted on github under kubevirt organization. You can very easily deploy it on your k8s by running following commands:
+
+```bash
+$ kubectl apply -f https://github.com/kubevirt/vm-import-operator/releases/download/v0.1.0/namespace.yaml
+$ kubectl apply -f https://github.com/kubevirt/vm-import-operator/releases/download/v0.1.0/operator.yaml
+$ kubectl apply -f https://github.com/kubevirt/vm-import-operator/releases/download/v0.0.4/vmimportconfig_cr.yaml
```
Comment (pkliczewski) [inline _posts/2020-08-06-Import-VM-from-oVirt.md]: Let's keep consistent versioning here and default to stable `v0.1.0`

File: _posts/2020-08-06-Import-VM-from-oVirt.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+      MIIEMjCCAxqgAwIBAgICEAAwDQYJKoZIhvcNAQELBQAwbDELMAkGA1UEBhMCVVMxJDAiBgNVBAoM
+      ....
+      fFyt91ClrUtTE707IFnYdQQUiZ4zI0q+6pmw6+xx8mH5k8Ad6D71pF718xCM1NiBx/Cusg==
+      -----END CERTIFICATE-----
+```
+
+Another step to initiate the import is creating the mappings. The mappings has three categories - storage mapping, disk mapping and network mapping. For storage mapping user can define which oVirt storage domain will be mapped to which storage class. Disk mapping can override the storage mapping for specific disks. The network mappings map oVirt's network to the kubernetes network. So here an simple example of mapping:
+
+```yaml
+apiVersion: v2v.kubevirt.io/v1alpha1
```
Comment (pkliczewski) [inline _posts/2020-08-06-Import-VM-from-oVirt.md:99]: I would add that it is possible to use internal mapping.

**Review by pkliczewski — COMMENTED** [review]
I think we can information about import rules and what is supported. 

Overall looks good.

**Comment by pkliczewski** [comment]
@codificat please review

**Comment by pkliczewski** [comment]
@mazzystr @chenyosef FYI

**Comment by mazzystr** [comment]
Please cancel this pull and recreate against upstream repo https://github.com/kubevirt/kubevirt.github.io branch source.  You might be able to modify it.  Edit this pull and see if that will work.  If it won't work just recreate it.

This update will need to be processed by CI before being allowed to be merged.

Thanks!

**Comment by machacekondra** [comment]
@mazzystr Ok thanks, I just wanted to make sure the blog is OK here privately, before pushing it to kubevirt repo. I will close this PR once we thing the structure is OK.

**Comment by machacekondra** [comment]
> I think we can information about import rules and what is supported.

I've added the info about rules. What do you mean by `supported`, to list all features, we don't block?

**Comment by pkliczewski** [comment]
> I've added the info about rules. What do you mean by `supported`, to list all features, we don't block?

I meant guests supported as specified [here](https://access.redhat.com/articles/4234591).

**Comment by pkliczewski** [comment]
Looks good to me

**Review by pkliczewski — APPROVED** [review]

**Pull request closed without merging**